components/Sections/Activity/Activity.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import prisma, { Prisma } from "@/lib/prisma";
+import { WYSIWYG } from "@/components/Shared/WYSIWYGComponent";
+import style from "./Activities.module.css";
```

Comment:
Here you import `./Activities.module.css`, but the actual file name is `./Activity.module.css`.